### PR TITLE
dismissInAppNotification support on android.

### PR DESCRIPTION
**Pull request by Royce**
Add .dismissInAppNotification on android.

- Separate hide and show animations.
- Implement a little bit of throttling around successive calls to .showInAppNotification

### CODE REVIEW

File: android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java [hunk 1]
```diff
@@ -28,33 +26,26 @@ public void animate() {
         animator.start();
     }
 
-    private Animator createAnimator(View view) {
+    private Animator createAnimator(View view, boolean show) {
```
Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java]: This is just nitpicking, but we can avoid the boolean and simply have two public methods:
`createShowAnimator(View view)`
`createHideAnimator(View view)`

**Comment by Royce** [comment]
@guyca That method (createAnimator) wasn't really required, so I removed it (and rebased), but I still pass the "show" boolean to the PeekingAnimator ctor.

Alternatively, I could make it more like https://github.com/wix/react-native-navigation/blob/master/android/app/src/main/java/com/reactnativenavigation/animation/VisibilityAnimator.java#L43

**Comment by varungupta85** [comment]
@Royce Thanks for the PR. Would the behavior for `showInAppNotification` be changed a little bit for Android after this PR? Before this PR, the in-app notification would appear and disappear shortly. There was no control to keep it there and dismiss it when I want or dismiss it after a set interval. With the addition of `dismissInAppNotification` on Android, would I be able to keep the in-app notification in the window till I explicitly call the dismiss method?

**Comment by Royce** [comment]
I'll be doing a few more PRs soon to address things like that.

**Pull request merged**